.travis.yml

```diff
@@ -25,7 +25,7 @@ matrix:
     - env: EMACS_VERSION=snapshot
 
 before_install:
-  - wget "https://raw.githubusercontent.com/vermiculus/emake.el/${EMAKE_SHA1}/emake.mk"
+  - make emake.mk
```

Comment:
If you include an emake.mk target in the same makefile that includes emake.mk, that will always create an error the first time the makefile is run.